site/app/controllers/GlobalController.php

```diff
@@ -19,7 +19,6 @@ public function header() {
                 'csrf_token' => $this->core->getCsrfToken()
             ]);
         }, $wrapper_files);
-
```

Comment:
This change isn't related to the rest of your PR and should be reverted.

Reply:
Are you talking about the space I don't think I actually changed any code either way I put the space back if that's what this was about.